### DISCUSSION TITLE
Custom Shortcuts Not Showing Even when Added

### DISCUSSION THREAD

**Discussion by Jonlb21** [Anze/KeyCluCask#84] [Q&A]
Hello, 

I was hoping to get some help with custom shortcuts not showing. In screenshot 1, you can see my custom hotkeys for my different Chrome extensions, but they don't show in Keyclu. Is Keyclu smart enough to automatically update the hotkeys when they are adjusted in app? It seems like when I change the hotkeys in any app, Keyclu still shows still shows the original defaults. 

I also use the Custom Menu Shortcuts app that integrates with KeyClu and lets you change the hotkeys in the KeyClu menu. But for some reason, when I add my custom hotkeys in Custom Menu Shortcuts they aren't showing in KeyClu for that app (see screenshot). I know KeyClu has an add shortcut option in the menu, but it doesn't make sense to use that version because if you change hotkeys via the KeyClu menu it redirects you to Custom Menu Shortcuts where the changes are made and saved there. So, why would would I want or need to manually add them to both Custom Menu Shortcuts and the Custom Shortcut Menu in setting inside of KeyClu. Is there a way to have the apps sync that data? 

![CleanShot 2024-07-01 at 05 00 41@2x](https://github.com/Anze/KeyCluCask/assets/161039460/7f79ac47-505a-4608-a8b8-fd99467b303d)
![CleanShot 2024-07-01 at 05 02 02@2x](https://github.com/Anze/KeyCluCask/assets/161039460/49c42634-9a4a-4609-bbe6-bfe0f66c670b)
![CleanShot 2024-07-01 at 05 03 15@2x](https://github.com/Anze/KeyCluCask/assets/161039460/03ab6260-d5e7-4b00-afcc-d32e25335a00)

**Comment by Anze** [comment]
Hey [Jonlb21](https://github.com/Jonlb21),

Thank you for your interest in KeyClu. And thanks for the screenshots, it helps a lot!

KeyClu is designed to show application shortcuts listed in the App's menu. It can't display the hotkeys you mentioned on common basis. To address this, was created the "My Shortcuts" section.

While CustomShortcuts app helps you organize changes to the App's menu shortcuts, but it doesn't handle hotkeys. The "My Shortcuts" section in KeyClu lets you list hotkeys not present in the App's menu. For example, Finder app doesn't list the option to toggle show/hide dot files in the menu, but the `shift`+`cmd`+`.` hotkey does the job. This can be listed in the "My Shortcuts" section.

Many apps, like Skype, don't list all their options in the menu. There's a repository called [KeyCluExtensions](https://github.com/Anze/KeyCluExtensions) that covers some apps with "missing" hotkeys. Those hotkeys added to "My Shortcut" section.

Keep in mind that some apps, like "DaVinci Resolve" define their menus in a way that's difficult to handle. I haven't found a workaround for this case yet.
